service/api.go

```diff
@@ -667,14 +667,15 @@ func (as *APIServer) validateProjectConfig(w http.ResponseWriter, r *http.Reques
 		errs = append(errs, validationErr)
 	}
 
+	errs = append(errs, validator.CheckProjectErrors(project, input.IncludeLong)...)
+	if len(errs) > 0 && input.Quiet {
+		errs = errs.AtLevel(validator.Error)
+		gimlet.WriteJSONError(w, errs)
+		return
+	}
 	errs = append(errs, validator.CheckYamlStrict(input.ProjectYaml)...)
```

Comment:
Should we move CheckYamlStrict to be part of CheckProjectWarnings, since it's just a warning?